content/kubermatic/master/tutorials_howtos/external_clusters/_index.en.md

```diff
@@ -28,10 +29,42 @@ To add a new external cluster go to Clusters -> External Clusters and press `Add
 
 ![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
-Select the Kubernetes clod provider. You can add the following external clusters:
+Select the Kubernetes cloud provider. You can add the following external clusters:
 
   - [GKE]({{< ref "./gke" >}})
   - [AKS]({{< ref "./aks" >}})
   - [EKS]({{< ref "./eks" >}})
-  
-## Connect existing cluster
+
+## Connect Existing Cluster
+
+To connect a cluster from any provider, click on `Any Provider` and provide the cluster name and kubeconfig.
+
+![Connect Cluster](/img/kubermatic/master/tutorials/external_clusters/connect.png "Connect Cluster")
+
+![Provide Kubeconfig](/img/kubermatic/master/tutorials/external_clusters/custom_cluster_credentials.png "Provide Kubeconfig")
+
+You can then see the details of the cluster.
+
+![Custom Cluster](/img/kubermatic/master/tutorials/external_clusters/custom_details.png "Custom Cluster")
+
+## Cluster State
+
+You can view the current state of your cluster by hovering the cursor over the small circle on the left of the cluster name.
+
+Provisioning state depicts that the cluster is gettig Created:
```

Comment:
s/gettig/getting
s/Created/created

Reply:
By the way, it is really getting created? We are dealing with already existing clusters.

Reply:
Okay, then just fix the words mentioned above.

Reply:
done